### PR TITLE
Update ExplosiveArrows_tombailey94.js

### PR DESCRIPTION
**Pull request by Samueljh1**
Fixed Crashing and Simplified! :)

### CODE REVIEW

**Comment by Samueljh1** [comment]
Oh. I forgot to do Entity.remove(ent); after Level.explode()!! Lol

File: ExplosiveArrows_tombailey94.js [hunk 1]
```diff
@@ -1,34 +1,28 @@
-// ExplosiveArrows by tombailey94
+// ExplosiveArrows by tombailey94 & Samueljh1
```
Comment (PEMapModder) [inline ExplosiveArrows_tombailey94.js]: No >_<

Comment (Samueljh1) [inline ExplosiveArrows_tombailey94.js]: xD

File: ExplosiveArrows_tombailey94.js [hunk 2]
```diff
@@ -1,34 +1,30 @@
...
 // version 0.0.1
 
-var Entities = [];
-var counter = 0;
+ent = null;
```
Comment (tombailey) [inline ExplosiveArrows_tombailey94.js]: Use var to make variable declaration explicit?

**Pull request merged**